### ISSUE TITLE
project-sidebar doesn't recognize project folders

### ISSUE THREAD

**Issue by bobrocke** [bripkens/project-sidebar#6]
If Atom has been used to add project folders to the tree view, project-sidebar doesn't recognize that and re-open those folders in the tree view.

**Comment by MattKaiserIO** [comment]
If I understand what you're saying I think all you need to do is perform the "save project" action again via project-manager. That way it will update the details in project-manager's projects.cson file with the new folders you added. Since project-sidebar pulls the list of projects from that projects.cson file it should now load all the folders you've associated with that project. Additionally, after doing the "save project", I had to toggle project-sidebar off and then back on for it to use the updated version of projects.cson (at least without exiting Atom).

**Comment by bobrocke** [comment]
Yes, that's what's happening. I can save manually, but was hoping there could be a hook that would do it automatically, as does Sublime Text.

**Comment by bripkens** [comment]
@bobrocke: project-sidebar is based on the project-manager project. As long as project-manager does not support this feature, project-sidebar won't support it either.

The only alternative would be to separate project-sidebar from the project-manager settings file.
